test/hls/hls_live_unit.js

```diff
@@ -921,6 +921,57 @@ describe('HlsParser live', () => {
             manifest.variants[1].video, [ref4]);
       });
 
+      describe('when ignoreManifestProgramDateTime is set', () => {
+        const config = shaka.util.PlayerConfiguration.createDefault().manifest;
+        config.hls.ignoreManifestProgramDateTime = true;
+
+        it('does not reset segment times when switching', async () => {
+          parser.configure(config);
+
+          const ref1 = makeReference(
+              'test:/main.mp4', 0, 2, /* syncTime= */ null);
+          const ref2 = makeReference(
+              'test:/main2.mp4', 2, 4, /* syncTime= */ null);
+
+          const secondVariant = [
+            '#EXT-X-STREAM-INF:BANDWIDTH=300,CODECS="avc1",',
+            'RESOLUTION=1200x940,FRAME-RATE=60\n',
+            'video2',
+          ].join('');
+          const masterWithTwoVariants = master + secondVariant;
+          configureNetEngineForInitialManifest(masterWithTwoVariants,
+              mediaWithAdditionalSegment, mediaWithAdditionalSegment2);
+
+          const manifest = await parser.start('test:/master', playerInterface);
+          await manifest.variants[0].video.createSegmentIndex();
+          ManifestParser.verifySegmentIndex(
+              manifest.variants[0].video, [ref1, ref2]);
+          expect(manifest.variants[1].video.segmentIndex).toBeNull();
+
+          // Update.
+          fakeNetEngine
+              .setResponseText('test:/video', mediaWithRemovedSegment)
+              .setResponseText('test:/video2', mediaWithRemovedSegment2);
+          await delayForUpdatePeriod();
+
+          // Before the switch, we know the earliest start time is 0, at
+          // EXT-X-MEDIA-SEQUENCE of 0.
+          expect(0).toBe(
```

Comment:
This (and line 971) should put the expected value inside `toBe` and the actual value inside `expect`.
If you do it this way, the error messages Jasmine prints on failure will be confusing.

Reply:
Agreed, done.